binancial/features/__init__.py

```
@@ -1 +1,5 @@
+from .datetime_related import datetime_related
```

Ruff (0.8.2)

`.datetime_related.datetime_related` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)